packages/shared/lib/auxiliary/notification/types/notification-type.type.ts

```diff
@@ -0,0 +1 @@
+export type NotificationType = 'info' | 'success' | 'warning' | 'error'
```

Comment:
Ideally this would be an `enum`, but didn't wanted to keep the scope of this PR purely about code organization.